scylla-gdb.py

```diff
@@ -5342,6 +5342,67 @@ def unqualify(name):
             unqualify(cfg_liveness)))
 
 
+class scylla_range_tombstones(gdb.Command):
+    """Prints and validates range tombstones in a given container.
+
+    Currently supported containers:
+        - mutation_partition
+
+    Example:
+
+        (gdb) scylla range-tombstones $mp
+        {
+          start: ['a', 'b'],
+          kind: bound_kind::excl_start,
+          end: ['a', 'b'],
+          kind: bound_kind::incl_end,
+          t: {timestamp = 1672546889091665, deletion_time = {__d = {__r = 1672546889}}}
+        }
+        {
+          start: ['a', 'b'],
+          kind: bound_kind::excl_start,
+          end: ['a', 'c']
+          kind: bound_kind::incl_end,
+          t: {timestamp = 1673731764010123, deletion_time = {__d = {__r = 1673731764}}}
+        }
+
+    """
+
+    def __init__(self):
+        gdb.Command.__init__(self, 'scylla range-tombstones', gdb.COMMAND_USER, gdb.COMPLETE_COMMAND)
+
+    """value is an instance of clustering_key"""
+    def parse_clustering(self, value):
+        return get_current_schema().parse_clustering_key(value)
```

Comment:
If no schema is set this should print a warning and tell the user to use `scylla set_schema` to set it first.

Reply:
It tells him that in the exception message:

```
def get_current_schema():
    if not current_schema:
        raise Exception('Current schema not set. Use \"scylla set-schema\" to set it.')
    return current_schema

```